client/coniksclient/internal/cmd/lookup.go

```diff
@@ -22,6 +22,8 @@ var lookupCmd = &cobra.Command{
 			cmd.Usage()
 			return
 		}
+		bindings := loadKeys(conf.KeyStoragePath)
```

Comment:
I gave a shot in 398dc3e124711cd4aceb2e1b3bbe528e8a18c9e5. Not sure how you feel about https://github.com/coniks-sys/coniks-go/pull/137/files#diff-8eead7b3ac1a29e7ad9e067fb3d9aaf8R59?